lib/gooddata/helpers/global_helpers_params.rb

```diff
@@ -52,9 +52,12 @@ def encode_hidden_params(params)
         encode_params(params, ENCODED_HIDDEN_PARAMS_KEY)
       end
 
-      # Decodes params as they came from the platform
+      # Decodes params as they came from the platform. It also replace reference parameters ${} in gd_encoded_params (if any) by actual value
       # The "data" key is supposed to be json and it's parsed - if this
-      def decode_params(params)
+      # @params Parameter hash need to be decoded
+      # @resolve_reference_params Resolve reference parameters in gd_encoded_params or not
+      # @return [Hash] Decoded parameters
+      def decode_params(params, resolve_reference_params = true)
```

Comment:
please use `options = { resolve_reference_params: true }` for readability in usage